CHANGELOG.md

```diff
@@ -1,9 +1,8 @@
 # 3.3.2 (2020-03-16)
 
-- Fixed the autocomplete component to properly highlight results after removing search text. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
-- Fixed the autocomplete component to properly highlight results when only one character is supplied. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
-- Fixed the country field component to properly fire the `selectedCountryChange` event when used outside of a form. [#63](https://github.com/blackbaud/skyux-lookup/pull/63)
-- Fixed the lookup component to correctly validate required values within a template-driven form. [#64](https://github.com/blackbaud/skyux-lookup/pull/64)
+- Fixed the autocomplete component to highlight results after removing search text and after only one character is supplied. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
+- Fixed an issue where the country field component only fired the `selectedCountryChange` event when the field was on a form. [#63](https://github.com/blackbaud/skyux-lookup/pull/63)
```

Comment:
This one isn't fully correct. It was firing the event outside of a form - it just wasn't firing the very first time that it should have. That's why I had the `properly` on the old version. Not sure how we would want to word that.

Reply:
Fixed an issue where the country field component only fired the `selectedCountryChange` event when the field was on a form. 

Fixed the country field component to properly fire the selectedCountryChange event when used outside of a form.

Reply:
Sorry. I missed that nuance when looking at the issues. How about:

`Fixed the country field component so that the selectedCountryChange event fires every time a country is selected.`